x/incentive/types/params.go

```diff
@@ -0,0 +1,166 @@
+package types
+
+import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"gopkg.in/yaml.v3"
+)
+
+const blocksPerDay = 86400 / 5
+
+var _ paramtypes.ParamSet = &Params{}
+
+var (
+	KeyMaxUnbondings           = []byte("MaxUnbondings")
+	KeyUnbondingDurationLong   = []byte("KeyUnbondingDurationLong")
+	KeyUnbondingDurationMiddle = []byte("KeyUnbondingDurationMiddle")
+	KeyUnbondingDurationShort  = []byte("KeyUnbondingDurationShort")
+	KeyTierWeightMiddle        = []byte("KeyTierWeightMiddle")
+	KeyTierWeightShort         = []byte("KeyTierWeightShort")
+)
+
+var (
+	defaultMaxUnbondings           = uint32(20)
+	defaultUnbondingDurationShort  = uint64(blocksPerDay)
+	defaultUnbondingDurationMiddle = uint64(blocksPerDay * 7)
+	defaultUnbondingDurationLong   = uint64(blocksPerDay * 14)
+	defaultTierWeightShort         = sdk.MustNewDecFromStr("0.5")
+	defaultTierWeightMiddle        = sdk.MustNewDecFromStr("0.8")
+)
+
+func NewParams() Params {
+	return Params{}
+}
+
+// ParamSetPairs implements the ParamSet interface and returns all the key/value
+// pairs pairs of x/incentive module's parameters.
+func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
```

Comment:
This file should be removed, and module params should be used (managed directly by module keeper and the new x/gov authority) rather than the paramspace.

https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-046-module-params.md#backwards-compatibility

Reply:
Added `MsgGovSetParams`